benchconnect/README.md

```diff
@@ -18,6 +18,28 @@ With [pipx](https://pypa.github.io/pipx/):
 pipx install benchconnect@git+https://github.com/conbench/conbench.git@main#subdirectory=benchconnect
 ```
 
+## Environment variables
+
+benchconnect relies on a number of environment variables to obtain various
+metadata:
+
+- `CONBENCH_URL`: Required. The URL of the Conbench server without a trailing
+slash, e.g. `https://conbench.example.com`
+- `CONBENCH_EMAIL`: The email to use for Conbench login. Only required if the
+server is private.
+- `CONBENCH_PASSWORD`: The password to use for Conbench login. Only required
+if the server is private.
+- `CONBENCH_PROJECT_REPOSITORY`: The repository name (in the format `org/repo`) or the
+URL (in the format `https://github.com/org/repo`)
+- `CONBENCH_PROJECT_PR_NUMBER`: [recommended] The number of the GitHub pull request that
+is running this benchmark, or ``None`` if for a run on the default branch
```

Comment:
nit: maybe instead of None here (which could result in `CONBENCH_PROJECT_PR_NUMBER="None"`) say something like "do not supply this if it's for a run on the default branch"